umbra-js/test/DomainService.test.ts

```diff
@@ -1,22 +1,26 @@
-const { provider } = require('@openzeppelin/test-environment');
-const { Resolution, Eip1993Factories } = require('@unstoppabledomains/resolution');
-const chai = require('chai');
-const DomainService = require('../classes/DomainService');
+import { DomainService } from '../src/classes/DomainService';
+import { provider } from '@openzeppelin/test-environment';
+import { Resolution, Eip1993Factories } from '@unstoppabledomains/resolution';
+import chai from 'chai';
+import constants from '../src/constants.json';
 
-const constants = require('../constants.json');
-const { CNS_REGISTRY } = constants;
+import type { Web3Version0Provider } from '@unstoppabledomains/resolution';
+import { ExternalProvider } from '../src/types';
 
+const { CNS_REGISTRY } = constants;
 const { expect } = chai;
 
 const resolution = new Resolution({
   blockchain: {
     cns: {
-      provider: Eip1993Factories.fromWeb3Version0Provider(provider),
+      provider: Eip1993Factories.fromWeb3Version0Provider(
+        (provider as unknown) as Web3Version0Provider
+      ),
       registry: CNS_REGISTRY,
     },
   },
 });
-const domainService = new DomainService(provider, resolution);
+const domainService = new DomainService((provider as unknown) as ExternalProvider, resolution);
```

Comment:
Same questions as with the provider above, what's going on here?

Reply:
Same as above